.buildkite/pipeline.yml

```diff
@@ -283,9 +283,315 @@ steps:
         agents:
           slurm_mem: 20GB
 
-      - label: ":balloon: Compressible single column EDMF Bomex"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config column --FLOAT_TYPE Float64 --hyperdiff false --moist equil --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --turbconv edmf --turbconv_case Bomex --dt_save_to_sol 5mins --z_elem 60 --z_stretch false --z_max 3e3 --job_id compressible_edmf_bomex --dt 1secs --t_end 6hours --regression_test true --apply_limiter false --debugging_tc true --dt_save_to_disk 6hours"
-        artifact_paths: "compressible_edmf_bomex/*"
+      - label: ":balloon: Compressible single column EDMF Bomex (Newton's method w/ explicit EDMF, 1s dt)"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config column --FLOAT_TYPE Float64 --hyperdiff false --moist equil --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --turbconv edmf --turbconv_case Bomex --dt_save_to_sol 5mins --z_elem 60 --z_stretch false --z_max 3e3 --job_id compressible_edmf_bomex --output_dir compressible_edmf_bomex_1s --dt 1secs --t_end 6hours --regression_test true --apply_limiter false --debugging_tc true --dt_save_to_disk 6hours"
```

Comment:
How many cases we want to add to the CI?

Do you think we can add the setup where we compare the new jobs (different JFNK options) with the current best performing one?

Reply:
I'm thinking we should just have 3 jobs for compressible bomex (all at their respective maximum dt): regular Newton's method with explicit EDMF, JFNK with explicit EDMF, and JFNK with implicit EDMF (as implicit as we can make it).